tests/cases/php/ternary-useless

```diff
@@ -0,0 +1,29 @@
+---DESCRIPTION---
+Ternary expressions must not be used unnecessarily
+---CONTENTS---
+<?php
+
+declare(strict_types=1);
+
+$foo = function () : bool {
+    return true ? true : false;
+};
+
+$bar = function () : bool {
+    return true === false ? false : true;
+};
+
+---FIXED---
+<?php
+
+declare(strict_types=1);
+
+$foo = function () : bool {
+    return true;
+};
+
+$bar = function () : bool {
+    return true !== false;
```

Comment:
Would be nice if this could be fixed automatically, otherwise PHPStan should pick it up (need to start using strict rules).